setup.py

```diff
@@ -47,6 +47,7 @@
             "nbconvert",
             "nbformat",
             "pre-commit",
+            "void-terminal==0.0.7",
```

Comment:
This dependency shouldn't be added here. There is no test using it.

Reply:
move to 

        "gui":[
            "void-terminal>=0.0.8",
        ]